data-reporting-service/investigation-service/src/main/java/gov/cdc/etldatapipeline/investigation/repository/model/dto/Investigation.java

```diff
@@ -0,0 +1,126 @@
+package gov.cdc.etldatapipeline.investigation.repository.model.dto;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.time.Instant;
+
+@Entity
+@Getter
```

Comment:
`@Data` can be used in the place of `@Getter` and `@Setter`

Reply:
Done